osf/models/storage.py

```diff
@@ -0,0 +1,10 @@
+
+from django.db import models
+
+from osf.models.base import BaseModel
+
+
+class ProviderAssetFile(BaseModel):
+    name = models.CharField(max_length=63)
+    file = models.FileField(upload_to='assets')
```

Comment:
Stupid minor; I personally don't like `file` as a name since it technically is a builtin... but we use it all over the place so whatev (https://docs.python.org/2/library/functions.html#file)

Reply:
I had the same mixed feeling about it, but I think it's only bad practice when assigning locals to builtins, as local assignment will override them but attrs/fields/methods will not. Some examples (in addition to yours) would be that any `TypedModel` has a [`type`](https://docs.python.org/2/library/functions.html#type) field, or that `Queryset` objects have [`filter`](https://docs.python.org/2/library/functions.html#filter) and [`all`](https://docs.python.org/2/library/functions.html#all) methods.

¯\\\_(ツ)_/¯